src/Entities/Categories.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+Class Categories{
```

Comment:
This class contains a single category, so the name should be singular. 

Also, please put the { on the next line.

Annnnnd Class should be class (little c)

Reply:
Done.